Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'rails', '~> 6.1.4'
 gem 'pg'
 gem 'puma', '~> 5.0'
 gem 'sass-rails', '>= 6'
-gem 'webpacker', '~> 5.0'
+gem 'webpacker', '~> 5.4.0'
```

Comment:
Was this change necessary for using Tailwind? (Asking to educate myself on dependencies 😃)

Reply:
The problem with the `5.0` version is that includes PostCSS 7 but the latest version of Tailwind requires version 8. So I had to follow this article: https://dev.to/davidteren/getting-started-with-rails-6-1-tailwindcss-jit-webpacker-postcss-8-25g7